infra/templates/azure-ase-app-service-single-region/docs/design.md

```diff
@@ -0,0 +1,161 @@
+# Deploying App Service into ASE 
```

Comment:
can we rename this template -> `azure-isolated-paas-single-region` which is aligned to the recent `azure-simple` rename.